vignettes/release-checklists.qmd

```diff
@@ -51,10 +52,19 @@ Parts of this document is adapted for The Hubverse from [The Carpentries Develop
 
 ### Release to CRAN
 
-The release process for CRAN releases requires extra time and attention.
-
- - [ ] run `devtools::release()` and follow the prompts
- - [ ] after CRAN submission, add and commit the `CRAN_SUBMISSION` file
+The release process for CRAN releases requires extra time and attention. Make sure to familiarise yourself with [the steps to release to CRAN](#release-to-cran). A good place to start would be to 
+
+ - [ ] open a template release issue using `usethis::use_release_issue()` and follow the instructions.
+ - [ ] If this the first release on CRAN, make sure to:
+    - [ ] Add the CRAN installation command to the installation instructions in the `README.Rmd`.
+    - [ ] Add a CRAN badge to the `README.Rmd` using `usethis::use_cran_badge()`.
+    = [ ] Rebuild `README.md`.
+ - [ ] Use `devtools::check(remote = TRUE, manual = TRUE)` to check the package locally.
+ - [ ] Use `devtools::check_win_devel()` to check the package with the devel version of R on <https://win-builder.r-project.org/>.
+ - Record the results of checks and respond concisely to any NOTEs not addressed in the `cran-comments.md` file. The contents of this file are included with package submission to CRAN. See more about [using `cran-comments.md` to communicate with CRAN](https://r-pkgs.org/release.html#sec-release-cran-comments). If the package does not have a `cran-comments.md` file, you can create one using `usethis::use_cran_comments()`.
+ - [ ] run `devtools::release()` and follow the prompts. This will run through a bunch of confirmation checks and finally submit the package to CRAN using `devtools::submit_cran()`.
```

Comment:
Here is adding the items for removing the Remotes before CRAN submission. I realized that it's easier to do directly in R instead of faffing with command line BS 😮‍💨.

I also removed the "using `devtools::submit_cran()`" function reference since `devtools::release()` already calls it. 

````suggestion
 - [ ] Make sure all your changes are committed and pushed to your release branch.
 - [ ] Remove the `Remotes:` items in the DESCRIPTION with the [`desc`](https://r-lib.github.io/desc) package:
   ```r
   desc::desc()$del("Remotes")$write()
   ``` 
   (DO NOT COMMIT, you will restore this later with `git restore DESCRIPTION`).
 - [ ] run `devtools::release()` and follow the prompts. This will run through a bunch of confirmation checks and finally submit the package to CRAN.
 - [ ] run `system("git restore DESCRIPTION")` to restore the `Remotes:` field in the `DESCRIPTION` file.
````